jetbrains-core/src/software/aws/toolkits/jetbrains/utils/CompatibilityUtils.kt

```diff
@@ -23,4 +40,59 @@ object CompatibilityUtils {
     @JvmStatic
     fun createPopupBuilder(validationInfo: ValidationInfo, configurator: Consumer<JEditorPane>?): ComponentPopupBuilder? =
         ComponentValidator::class.staticFunctions.find { it.name == "createPopupBuilder" }?.call(validationInfo, configurator) as? ComponentPopupBuilder
+
+    /**
+     * Can be removed when min-version is 19.1 FIX_WHEN_MIN_IS_191
```

Comment:
All things that are actually `FIX_WHEN_MIN_IS_191` are currently `FIX_WHEN_MIN_IS_192` to have a single string to look for

Reply:
? the idea here would be that there's a common pattern string to look for when we move off a version ... not sure what you mean